integrations/pgsql.ini.tmpl

```diff
@@ -26,7 +26,8 @@ SSH_DOMAIN       = localhost
 HTTP_PORT        = 3002
 ROOT_URL         = http://localhost:3002/
 DISABLE_SSH      = false
-SSH_PORT         = 22
+SSH_LISTEN_HOST  = localhost
+SSH_PORT         = 2222
```

Comment:
Do not use same port for multiple database tests as they are run in parallel

Reply:
Good catch